README.md

```diff
@@ -1,80 +1,130 @@
 
-https://designpatterns.hackpad.com/GOV.UK-elements-feedback-sKrDyQxcfA2
 
-#### Code
+    // GOV.UK elements
+    @import "elements/helpers";
+    @import "elements/reset";
+    @import "elements/elements-typography";
+    @import "elements/layout";
 
-The code in GOV.UK Elements is built on top of the [GOV.UK front end toolkit](https://github.com/alphagov/govuk_frontend_toolkit).
+    @import "elements/forms";
+    @import "elements/forms/form-block-labels";
+    @import "elements/forms/form-date";
+    @import "elements/forms/form-validation";
 
-If you want to suggest changes or raise bugs on code from the toolkit, please do so through its repository, not this one.
+    @import "elements/tables";
+    @import "elements/lists";
+    @import "elements/details";
+    @import "elements/panels";
+    @import "elements/buttons";
+    @import "elements/icons";
 
-GOV.UK Elements code in use across live projects but not already in the toolkit should eventually be added to it. All contributions towards this goal are welcome and should be in the form of pull requests on the toolkit repository.
+    @import "elements/components";
+    @import "elements/breadcrumb";
 
-Examples of pull requests already raised to this end:
 
-- https://github.com/alphagov/govuk_frontend_toolkit/pull/100
-- https://github.com/alphagov/govuk_frontend_toolkit/pull/121
+Ignore the `/public/sass/elements-page.scss` files, these exist to style the page furniture of GOV.UK elements (for example, the HTML example boxes and colour swatches).
 
-### Navigating this repository
+## Running this site locally
 
-#### To view the Sass files:
+If you would like to clone the repository and run it locally,
+you will need [Node.js](http://nodejs.org/) (at least version v0.10.0).
 
-You can find the Sass files here:
+Clone this repository
 
-```
-/public/sass/
-```
+    git clone git@github.com:alphagov/govuk_elements.git
 
-```
-/public/sass/main.scss
-```
 
-The main.scss file shows how to import the Elements Sass files.
+Install the required node modules
 
-#### To view this site locally:
+    npm install
 
-If you would like to clone the repository and run it locally,
-you will need [Node.js](http://nodejs.org/) (at least version v0.10.0).
 
-* clone this repository
-* install the required node modules
-```
-npm install --production
-```
-* run the app
-```
-foreman start -p 3000
-```
-* go to [localhost:3000](http://localhost:3000) in your browser.
+Run the app
+
+    node start.js
+
+
+Go to [localhost:3000](http://localhost:3000) in your browser.
+
+### Running wraith to compare changes
+
+GOV.UK elements uses Wraith so that regressions can be easily spotted.
+Runs against all the main elements pages and some of the examples.
+
+#### Usage
+
+On master run:
+
+    wraith history config.yaml
+
+
+On local branch run
+
+    wraith latest config.yaml
+
+
+##  How to we want people to contribute and improve elements.
```

Comment:
There's a typo here. There's a Github pattern that puts contribution guidance in a file named `CONTRIBUTING.md`, eg https://github.com/alphagov/govuk_frontend_toolkit/blob/master/CONTRIBUTING.md